src/views/SearchCommons.vue

```diff
@@ -45,7 +45,9 @@
             return {
               id: index + 1,
               img: item.thumb,
-              text: item.desc
+              text: item.desc,
+              imgTitle: item.title,
+              attribution: null
```

Comment:
when fetching the commons, the attribution already existed in the api (if not then we can add to the api), 
can you use that directly instead of fetching imageinfo for each.

Reply:
Nice, good call on saving us those api trips and this way we don't need the `forEach` in Story.vue